components/org.wso2.carbon.identity.api.server.idv.provider/org.wso2.carbon.identity.api.server.idv.provider.v1/src/gen/java/org/wso2/carbon/identity/api/server/idv/provider/v1/model/IdVProviderRequest.java

```diff
@@ -53,11 +53,9 @@ public IdVProviderRequest type(String type) {
         return this;
     }
     
-    @ApiModelProperty(example = "ONFIDO", required = true, value = "")
-    @JsonProperty("Type")
+    @ApiModelProperty(example = "ONFIDO", value = "")
```

Comment:
Once you build after incorporating the change [1], this change should not be there.

[1] https://github.com/wso2/identity-api-server/pull/669/files#r1779901382

Reply:
Fixed with 65c27fd31bbdca159c03b7d0cea0349fa187172b